File: 1045.py
a, b, c = map(float, input().split())

if a < b:
    a, b = b, a
if b < c:
    b, c = c, b
if a < b:
    a, b = b, a

if a >= b + c:
    print("NAO FORMA TRIANGULO")
elif a**2 == b**2 + c**2:
    print("TRIANGULO RETANGULO")
elif a**2 > b**2 + c**2:
    print("TRIANGULO OBTUSANGULO")
elif a**2 < b**2 + c**2:
    print("TRIANGULO ACUTANGULO")
if a == b == c:
    print("TRIANGULO EQUILATERO")
elif a == b or b == c or a == c:
    print("TRIANGULO ISOSCELES")

File: Dart/1045.dart
import 'dart:io';

void main() {
  var n = stdin.readLineSync().split(' ').map(double.parse).toList();
  if (n[0] < n[1]) {
    var temp = n[0];
    n[0] = n[1];
    n[1] = temp;
  }
  if (n[0] < n[2]) {
    var temp = n[0];
    n[0] = n[2];
    n[2] = temp;
  }
  if (n[1] < n[2]) {
    var temp = n[1];
    n[1] = n[2];
    n[2] = temp;
  }
  if (n[0] >= n[1] + n[2]) {
    print("NAO FORMA TRIANGULO");
  } else if (n[0] * n[0] == n[1] * n[1] + n[2] * n[2]) {
    print("TRIANGULO RETANGULO");
  } else if (n[0] * n[0] > n[1] * n[1] + n[2] * n[2]) {
    print("TRIANGULO OBTUSANGULO");
  } else if (n[0] * n[0] < n[1] * n[1] + n[2] * n[2]) {
    print("TRIANGULO ACUTANGULO");
  }
  if (n[0] == n[1] && n[1] == n[2]) {
    print("TRIANGULO EQUILATERO");
  } else if (n[0] == n[1] || n[1] == n[2] || n[0] == n[2]) {
    print("TRIANGULO ISOSCELES");
  }
}
